doc/lua_api.txt

```diff
@@ -6073,14 +6073,37 @@ object you are working with still exists.
     * Only affects formspecs shown after this is called.
 * `get_formspec_prepend(formspec)`: returns a formspec string.
 * `get_player_control()`: returns table with player pressed keys
-    * The table consists of fields with boolean value representing the pressed
-      keys, the fields are jump, right, left, LMB, RMB, sneak, aux1, down, up.
-    * example: `{jump=false, right=true, left=false, LMB=false, RMB=false,
-      sneak=true, aux1=false, down=false, up=false}`
+    * The table consists of fields with boolean values representing the pressed
+      keys. Example:
```

Comment:
Is this example required? Just listing the table keys like before should be enough, isn't it?

Reply:
Personally, I think listing the keys would suffice and the example is required at all. Shall I remove it?

Reply:
Done